src/java/picard/analysis/directed/GcBiasMetricsCollector.java

```diff
@@ -215,6 +224,34 @@ else if (group.equals("Library")) {
     }
 
     /////////////////////////////////////////////////////////////////////////////
+    // Calculates the normalized coverage over each gc content quintile
+    /////////////////////////////////////////////////////////////////////////////
+    private ArrayList<Double> calculateGcNormCoverage(final Double meanReadsPerWindow, final int[] readsByGc) {
+        final ArrayList<Double> gcNormCovIntervals = new ArrayList<Double>();
+        int readStartsTotal = 0;
+        int windowsTotal = 0;
+
+        for (int i = 0; i < windowsByGc.length; i++) {
```

Comment:
not sure this function is working properly...could you add a unit test for it specifically?